src/psyclone/psyir/backend/sympy_writer.py

```diff
@@ -319,7 +331,8 @@ def _create_type_map(self, list_of_expressions, identical_variables=None):
                 unique_sym = self._symbol_table.new_symbol(name, tag=name)
                 # Test if an array or an array expression is used:
                 if not ref.is_array:
-                    self._sympy_type_map[unique_sym.name] = Symbol(name)
+                    self._sympy_type_map[unique_sym.name] = Symbol(
```

Comment:
This Symbol instance got me confused, maybe we could change the import so that we do `sympy.Symbol(` here?

Reply:
Yes, good idea.